frontend/src/__tests__/cypress/cypress/pages/modelRegistry/modelVersionDetails.ts

```diff
@@ -32,6 +32,23 @@ class ModelVersionDetails {
     return cy.findByTestId('pipeline-run-link');
   }
 
+  shouldNotHavePipelineRunLink() {
+    this.findPipelineRunLink().find('a').should('not.exist');
+    return this;
+  }
+
+  findInfoPopover() {
+    return cy.findByTestId('project-info');
+  }
+
+  shouldHaveProjectInfoText() {
+    cy.findByTestId('pipeline-project-popover').should(
+      'contain.text',
+      'Your administrator might be:',
```

Comment:
I think you should rather check for `You don't have access to the source run.` coz it's more relevant ?